.jenkins

```diff
@@ -60,14 +50,11 @@ pipeline {
       }
       steps {
         script {
-          docker.image('vaporio/jenkins-agent-golang:latest').inside('-v /var/run/docker.sock:/var/run/docker.sock -v /usr/bin/docker:/usr/bin/docker -e GITHUB_TOKEN=$GITHUB_TOKEN --group-add docker') {
```

Comment:
So, this stage isn't being run in a golang container, so does that mean that it's running on the "golang-next" agent? If so, I'm just wondering what the difference is between running in a golang container vs. the golang-next agent, and when one should be preferred over the other

Reply:
you shouldn't ever have to use the docker.image script hack now that this runs in [this agent](https://github.com/vapor-ware/jenkins-agents/blob/master/golang/Dockerfile) which has runtime tooling/support for what this container was doing on the build-master previously:

Not pictured: the docker socket / binary is volume mounted into the container :) 

There may be some bugs in this pipeline but i'm free to help sort them out.

Reply:
Good to know, thanks! I realize my question wasn't totally clear though, what I meant to ask was why the other stages are being run in `container('golang'){` where this is just being run on the golang agent directly

Reply:
laziness, and it should be updated. :smile:

Reply:
Bumped, and shuffled those credentials exposure around a bit based on the above comment. :+1: thanks for keeping me honest ;D